cmd/k3s/main.go

```diff
@@ -46,8 +47,8 @@ func main() {
 	app := cmds.NewApp()
 	app.EnableBashCompletion = true
 	app.Commands = []cli.Command{
-		cmds.NewServerCommand(internalCLIAction(version.Program+"-server", dataDir, os.Args)),
-		cmds.NewAgentCommand(internalCLIAction(version.Program+"-agent", dataDir, os.Args)),
+		cmds.NewServerCommand(internalCLIAction(version.Program+"-server"+version.ProgramPostfix, dataDir, os.Args)),
+		cmds.NewAgentCommand(internalCLIAction(version.Program+"-agent"+version.ProgramPostfix, dataDir, os.Args)),
```

Comment:
I don't think this belongs as an exported variable in the `version` package - it's not something that's going to be overridden at build time or in downstream projects.

Can we add main_linux.go/main_windows.go and just make it a package variable? It doesn't look like it's used anywhere else so it doesn't even need to be exported.